praesentia/static/praesentia.js

```diff
@@ -6,10 +6,30 @@ let html5QrcodeScanner = new Html5QrcodeScanner(
         qrbox: 250,
     });
 
-
 html5QrcodeScanner.render(onScanSuccess);
 
+const target = document.querySelector('body');
+
+target.addEventListener('paste', (event) => {
```

Comment:
I have been using jQuery throughout the file. I don't know whether or not to continue using jQuery. Or, for the sake of consistency, should we just use jQuery?

Reply:
I guess so, for consistency of this project. I can translate my code using jQuery